File: SecondYear/riababbar4_RiaBabbar_2226en1042_2/Week_1/while loop.py
def sum_of_even_odd_digits(n):
    even_sum = 0
    odd_sum = 0
    
    while n > 0:
        digit = n % 10
        if digit % 2 == 0:
            even_sum += digit
        else:
            odd_sum += digit
        n //= 10

    print(f"Sum of even digits: {even_sum}")
    print(f"Sum of odd digits: {odd_sum}")

File: SecondYear/ksh1710_KshitizAgarwal_2226cse1033_2/Week_1/while_loop.cpp
#include<iostream>
using namespace std;

int main() {
	int n,evenSum=0,oddSum=0,d;
	cin>>n;
	while(n!=0){
		d=n%10;
		n=n/10;
		if(d%2==0){
			evenSum+=d;
		}else{
			oddSum+=d;
		}
	}
	cout<<evenSum<<" "<<oddSum;
}
